.github/workflows/generate-baseline-profile.yml

```diff
@@ -69,10 +69,10 @@ jobs:
             --no-configuration-cache
 
       - name: Move & Rename Baseline Profile
-        run: mv -f macrobenchmark/forlago/build/intermediates/managed_device_android_test_additional_output/benchmark/pixel2Api31googleBenchmarkAndroidTest/BaselineProfileGenerator_generate-baseline-prof.txt  apps/forlago/src/main/baselineProfiles/baseline-prof.txt
+        run: mv -f macrobenchmark/forlago/build/outputs/managed_device_android_test_additional_output/benchmark/pixel2Api31googleBenchmark/BaselineProfileGenerator_generate-baseline-prof.txt  apps/forlago/src/main/baselineProfiles/baseline-prof.txt
```

Comment:
Now seems the file is generated again in the `outputs` folder 🫣 I don't get the reason...
![Screenshot 2024-12-04 at 11 44 52](https://github.com/user-attachments/assets/08f63126-5bc3-4e3d-887a-b106c4e48a4b)

Reply:
:open_mouth:

Reply:
It doesn't seem very reliable